libs/core/src/lib/toolbar/toolbar.component.ts

```diff
@@ -206,6 +217,14 @@ export class ToolbarComponent implements OnInit, AfterViewInit, OnDestroy, After
         this.buildComponentCssClass();
     }
 
+    /** @hidden */
+    ngAfterContentInit(): void {
+        if (this.titleComponent) {
+            this.titleComponent.elementRef?.nativeElement.classList.add('fd-toolbar__title');
```

Comment:
I think it should be done in `TitleComponent` itself. 

There we may get toolbar as an optional dependency and check if it's present - apply class. Also, this dependency should rely not on the implementation but on the injection token, see my comment above.

Reply:
It's a toolbar class so in my opinion makes more sense to happen here